libraries/AC_AutoTune/AC_AutoTune.cpp

```diff
@@ -218,119 +218,27 @@ bool AC_AutoTune::init_position_controller(void)
     return true;
 }
 
-const char *AC_AutoTune::level_issue_string() const
-{
-    switch (level_problem.issue) {
-    case LevelIssue::NONE:
-        return "None";
-    case LevelIssue::ANGLE_ROLL:
-        return "Angle(R)";
-    case LevelIssue::ANGLE_PITCH:
-        return "Angle(P)";
-    case LevelIssue::ANGLE_YAW:
-        return "Angle(Y)";
-    case LevelIssue::RATE_ROLL:
-        return "Rate(R)";
-    case LevelIssue::RATE_PITCH:
-        return "Rate(P)";
-    case LevelIssue::RATE_YAW:
-        return "Rate(Y)";
-    }
-    return "Bug";
-}
-
-void AC_AutoTune::send_step_string()
-{
-    if (pilot_override) {
```

Comment:
I think this one is interesting to keep

Reply:
We have another pilot override warning with separate timer

https://github.com/ArduPilot/ardupilot/blob/adff48c3ea332acece4445f50f6c1300df2dad3f/libraries/AC_AutoTune/AC_AutoTune.cpp#L387-L392